examples/cifar_simple/__init__.py

```diff
@@ -1,5 +1,10 @@
 # flake8: noqa
-from catalyst.dl import SupervisedRunner as Runner
+
+import experiments as exp
+
+from catalyst.dl import registry, SupervisedRunner as Runner
 
 from .experiment import Experiment
 from .model import SimpleNet
+
+registry.EXPERIMENTS.add_from_module(exp)
```

Comment:
PR look great, nevertheless, based on logic above
```python
if Experiment is None:
        experiment_params = config.get("experiment_params", {})
        experiment = experiment_params.get("experiment", "Experiment")
        Experiment = EXPERIMENTS.get(experiment)
```

this case will not start - we have predefined `from .experiment import Experiment`

could you please add new test to [catalyst/tests](https://github.com/catalyst-team/catalyst/tree/master/tests) and add it run to [tests.sh](https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_cv.sh) ?

I think, something like

```bash
################################  pipeline 16  ################################
echo 'pipeline 16'
EXPDIR=./tests/_tests_cv_classification_experiment_registry
LOGDIR=./tests/logs/_tests_cv_classification_experiment_registry
LOGFILE=${LOGDIR}/checkpoints/_metrics.json

PYTHONPATH=./examples:./catalyst:${PYTHONPATH} \
  python catalyst/dl/scripts/run.py \
  --expdir=${EXPDIR} \
  --config=${EXPDIR}/config1_simple_experiment.yml \
  --logdir=${LOGDIR} \
  --check

if [[ ! (-f "$LOGFILE" && -r "$LOGFILE") ]]; then
    echo "File $LOGFILE does not exist"
    exit 1
fi

cat $LOGFILE
echo 'pipeline 16'
python -c """
from catalyst import utils
metrics = utils.load_config('$LOGFILE')
# assert metrics['stage1.2']['loss'] < metrics['stage1.1']['loss']
assert metrics['stage1.2']['loss'] < 2.33
"""

rm -rf ${LOGDIR}
```

Reply:
@Scitator , Thanks for your comments. 
I added 3 related testcases for experiment registry. 
Please help me check it